src/IpScanner/Program.cs

```diff
@@ -5,17 +5,24 @@ namespace IpScanner;
 
 internal static class Program
 {
-    private static bool IsDisplayUnknownHost => false; // TODO: add ability for change from args
+    private static bool IsDisplayUnknownHost = true;
```

Comment:
please use false by default

Reply:
are you sure about this? I mean you are saying you adding -u will remove the the unknown host. it not it opposite what you saying?

Reply:
just to clarify:
- argument  `-u` => `unknown hosts` means display info about unknown hosts, so argument `-u` means that `IsDisplayUnknownHost=true`
- without `-u` - default behaver - should be as before, means `IsDisplayUnknownHost=false`

Reply:
but as you implement in your code IsDisplayUnknownHost=true means don't show the unknown hosts. and if it is false then show the unknown hosts.

Reply:
hmm, yes, now I see the issue with naming and using 👀 

ok, I will fix it later